pkg/pb/satellite.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package satellite;
+
+import "gogo.proto";
+
+service Satellite {
```

Comment:
Satellite is a composite service, so calling it `Satellite`  makes it confusing. Adding a general service makes it a gravity center, which can easily become a kitchen-sink - you can see the similar effect, because we now also needed `endpoints` package.

Also, I don't think a single service is necessary since protobuf should be able to connect to a composite of interfaces... not sure whether there's a capability out-of-the-box, but based on the RPC, it seems plausible.

The file health is about talking to pointerdb / metainfo. So it should probably be in one of those. Let's call this endpoint `Metainfo`, because we we'll need one soon anyways and seems more suitable.

Reply:
Changed to Metainfo

Reply:
I renamed everything called `metainfo` in `peer.go` to `pointerdb` as a result to reduce confusion